docs: fix frozen install help (#1167)
prefix-dev/pixi: docs/reference/cli.md

Which gets generated on `pixi add` or when you manually change the [manifest file](configuration.md) file and run `pixi install`.

##### Options
    /// Install the environment as defined in the lockfile, doesn't abort when lockfile isn't up-to-date with the manifest file.

- `--manifest-path <MANIFEST_PATH>`: the path to [manifest file](configuration.md), by default it searches for one in the parent directories.

> - `--frozen`: install the environment as defined in the lockfile, doesn't abort when lockfile isn't up-to-date with the manifest file. It can also be controlled by the `PIXI_FROZEN` environment variable (example: `PIXI_FROZEN=true`).

Reviewer: It would only abort with `--locked` right?
Without any flags it would update the lockfile.
Maybe this makes more sense then?
```suggestion
- `--frozen`: install the environment as defined in the lockfile, doesn't update `pixi.lock` if it isn't up-to-date with `pixi.toml`. It can also be controlled by the `PIXI_FROZEN` environment variable (example: `PIXI_FROZEN=true`).
```

Author: I changed it but not exactly the same way you suggest.

Reviewer: Your change looks good to me :)
